recipe/meta.yaml

```diff
@@ -11,8 +11,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
+  noarch: python
```

Comment:
It looks like this is now `noarch`. However we still have compilers in `requirements/build`. Should those be dropped or does this package still have architecture dependence?

Reply:
It needs no compiler. Maybe I put that into the closed PR instead.